configure.ac

```diff
@@ -242,7 +243,7 @@ AC_LINK_IFELSE([
 #endif
          ]])],
          AC_MSG_RESULT([assume yes (cross compiling)]),
-	 flag_finite_math_only=yes)
+      flag_finite_math_only=yes)
```

Comment:
This indent was not what I suggested before (it's one less level than I originally had)

Reply:
Was effectively `\t\x20` before, miscounted the level from the parens, should be one level deeper. Will update.